src/meson.build

```diff
@@ -367,6 +367,57 @@ executable(
   install_dir: daemon_dir
 )
 
+if libgbinder.found()
+  executable(
+    'fwupd-binder',
+    fwupdengine_rs,
+    fwupddaemon_rs,
+    plugins_hdr,
+    sources: [
+      daemon_loader_src,
+      'fu-daemon.c',
+      'fu-binder-daemon.c',
+    ],
```

Comment:
I don't really like the idea of having a second daemon that is binder vs dbus based.  Why can't we have the single daemon run with whatever is compiled in?  A single daemon could listen on both binder and dbus if both are compiled in...

This will make test coverage a lot easier too then.

Reply:
It's going to be very complicated  that way; binder and dbus are set up in very different ways.